### PR TITLE
RestartThenRepairNode: increase database up timeout to 4 hours

### PR DESCRIPTION
**Pull request by amoskong**
In one 1TB longevity, the database serivce on target node took about 3+ hours
to be up, original data was removed during the restart, the default 1 hour
isn't enough for streaming.

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def _kill_scylla_daemon(self):
 
         # Let's wait for the target Node to have their services re-started
         self.log.info('Waiting scylla services to be restarted after we killed them...')
-        self.target_node.wait_db_up()
+        self.target_node.wait_db_up(timeout=14400)
```
Comment (bentsi) [inline sdcm/nemesis.py]: please add comment/explain in commit messsage why did you add this timeout

Comment (amoskong) [reply sdcm/nemesis.py]: I had explain in current commitlog, isn't it clear?

https://github.com/scylladb/scylla-cluster-tests/pull/704/commits/8f66ba2b18396d0383577767e84dee8dce6e90df

Comment (bentsi) [reply sdcm/nemesis.py]: @amoskong oh sorry, I didn't see the full commit message :+1:

Comment (bentsi) [reply sdcm/nemesis.py]: anyway, we need to find a way how to make this more robust, like querying Scylla. I will open the task for that in Trello. 
@roydahan Do we need to port this to `branch-3.0`?

Comment (roydahan) [reply sdcm/nemesis.py]: Yes, please.

**Comment by roydahan** [comment]
please merge and backportto 3.0

**Comment by bentsi** [comment]
merged and backported to `branch-3.0`
